### ISSUE TITLE
Detected call of `lr_scheduler.step()` before `optimizer.step()`

### ISSUE THREAD

**Issue by zbwxp** [Alibaba-MIIL/ImageNet21K#47]
When run training, a warning appear:

    UserWarning: Detected call of `lr_scheduler.step()` before `optimizer.step()`. In PyTorch 1.1.0 and later, you should call them in 
    the opposite order: `optimizer.step()` before `lr_scheduler.step()`.  Failure to do this will result in PyTorch skipping the first 
    value of the learning rate schedule. See more details at https://pytorch.org/docs/stable/optim.html#how-to-adjust-learning-rate

Since the requirement of this code is torch 1.7.1, I assume the code can be affected by the warning. Should I reorder the call? or just ignore the warning?
